packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -15,54 +15,52 @@ import { Header } from '../partials';
 
 const MIN_LENGTH = 6;
 
-interface Props extends RouteComponentProps<{address: string}>, ThemeProps {
+interface Props extends RouteComponentProps<{ address: string }>, ThemeProps {
   className?: string;
 }
 
-function Export ({ className, match: { params: { address } } }: Props): React.ReactElement<Props> {
+function Export({
+  className,
+  match: {
+    params: { address }
+  }
+}: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const onAction = useContext(ActionContext);
   const [isBusy, setIsBusy] = useState(false);
   const [pass, setPass] = useState('');
   const [error, setError] = useState('');
 
-  const _goHome = useCallback(
-    () => onAction('/'),
-    [onAction]
-  );
+  const _goHome = useCallback(() => onAction('/'), [onAction]);
 
-  const onPassChange = useCallback(
-    (password: string) => {
-      setPass(password);
-      setError('');
-    }
-    , []);
-
-  const _onExportButtonClick = useCallback(
-    (): void => {
-      setIsBusy(true);
-
-      exportAccount(address, pass)
-        .then(({ exportedJson }) => {
-          const blob = new Blob([JSON.stringify(exportedJson)], { type: 'application/json; charset=utf-8' });
-
-          saveAs(blob, `${address}.json`);
-
-          onAction('/');
-        })
-        .catch((error: Error) => {
-          console.error(error);
-          setError(error.message);
-          setIsBusy(false);
-        });
-    },
-    [address, onAction, pass]
-  );
+  const onPassChange = useCallback((password: string) => {
+    setPass(password);
+    setError('');
+  }, []);
+
+  const _onExportButtonClick = useCallback((): void => {
+    setIsBusy(true);
+
+    exportAccount(address, pass)
+      .then(({ exportedJson }) => {
+        const blob = new Blob([JSON.stringify(exportedJson)], { type: 'application/json; charset=utf-8' });
+
+        saveAs(blob, `${address}.json`);
+
+        onAction('/');
+      })
+      .catch((error: Error) => {
+        console.error(error);
+        setError(error.message);
+        setIsBusy(false);
+      });
+  }, [address, onAction, pass]);
 
   return (
     <>
       <Header
         showBackArrow
+        showHelp
```

Comment:
Based on recent changes in naming conventions, if that's a boolean maybe `hasHelp` or `withHelp`?